toolbox/events/sbo.png.url

```diff
@@ -0,0 +1 @@
+http://www.spieleentwicklung-bodensee.de/info.txt
```

Comment:
Da fehlt ein "s" in der URL. Wer weiß wie lange unverschlüsselter traffic in der toolbox unverändert bleibt...

Reply:
wurde gefixt... sollte nun gehen

Reply:
sehr cool! danke!